frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { GraphElement } from '@patternfly/react-topology';
```

Comment:
main vendor size is increased because of this import most probably

Reply:
Figured out it was due to importing getResource util in the providers.ts file